diff -Naur GeographicLib-prefix/src/GeographicLib/tools/geographiclib-get-geoids.sh GeographicLib-prefix-patched/src/GeographicLib/tools/geographiclib-get-geoids.sh
--- GeographicLib-prefix/src/GeographicLib/tools/geographiclib-get-geoids.sh	2013-12-11 16:01:51.000000000 +0100
+++ GeographicLib-prefix-patched/src/GeographicLib/tools/geographiclib-get-geoids.sh	2014-08-26 23:18:36.000000000 +0200
@@ -103,7 +103,7 @@
 trap 'trap "" 0; test "$TEMP" && rm -rf "$TEMP"; exit 1' 1 2 3 9 15
 trap            'test "$TEMP" && rm -rf "$TEMP"'            0
 fi
-TEMP=`mktemp --tmpdir --quiet --directory $NAME-XXXXXXXX`
+TEMP=`mktemp -q -d $NAME-XXXXXXXX`
 
 if test -z "$TEMP" -o ! -d "$TEMP"; then
     echo Cannot create temporary directory 1>&2
diff -Naur GeographicLib-prefix/src/GeographicLib/tools/geographiclib-get-gravity.sh GeographicLib-prefix-patched/src/GeographicLib/tools/geographiclib-get-gravity.sh
--- GeographicLib-prefix/src/GeographicLib/tools/geographiclib-get-gravity.sh	2013-12-11 16:01:51.000000000 +0100
+++ GeographicLib-prefix-patched/src/GeographicLib/tools/geographiclib-get-gravity.sh	2014-08-26 23:18:36.000000000 +0200
@@ -93,7 +93,7 @@
 trap 'trap "" 0; test "$TEMP" && rm -rf "$TEMP"; exit 1' 1 2 3 9 15
 trap            'test "$TEMP" && rm -rf "$TEMP"'            0
 fi
-TEMP=`mktemp --tmpdir --quiet --directory $NAME-XXXXXXXX`
+TEMP=`mktemp -q -d $NAME-XXXXXXXX`
 
 if test -z "$TEMP" -o ! -d "$TEMP"; then
     echo Cannot create temporary directory 1>&2
diff -Naur GeographicLib-prefix/src/GeographicLib/tools/geographiclib-get-magnetic.sh GeographicLib-prefix-patched/src/GeographicLib/tools/geographiclib-get-magnetic.sh
--- GeographicLib-prefix/src/GeographicLib/tools/geographiclib-get-magnetic.sh	2013-12-11 16:01:51.000000000 +0100
+++ GeographicLib-prefix-patched/src/GeographicLib/tools/geographiclib-get-magnetic.sh	2014-08-26 23:18:36.000000000 +0200
@@ -91,7 +91,7 @@
 trap 'trap "" 0; test "$TEMP" && rm -rf "$TEMP"; exit 1' 1 2 3 9 15
 trap            'test "$TEMP" && rm -rf "$TEMP"'            0
 fi
-TEMP=`mktemp --tmpdir --quiet --directory $NAME-XXXXXXXX`
+TEMP=`mktemp -q -d $NAME-XXXXXXXX`
 
 if test -z "$TEMP" -o ! -d "$TEMP"; then
     echo Cannot create temporary directory 1>&2
